x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/reducer.ts

```diff
@@ -144,6 +144,9 @@ export const validateAdvancedEditor = (state: State): State => {
   const destinationIndexNameValid = isValidIndexName(destinationIndexName);
   const destinationIndexPatternTitleExists =
     state.indexPatternsMap[destinationIndexName] !== undefined;
+
+  const resultsFieldEmptyString = jobConfig?.dest?.results_field === '';
```

Comment:
Should probably do a `trim` on this before checking if it's empty. When I tried entering a string of whitespaces it's worth noting that no error was reported from the back-end, and the job seemed to stall in the writing of results phase, with these errors in the elasticsearch.log file
```
[2020-07-07T16:58:39,530][INFO ][o.e.x.m.d.p.AnalyticsResultProcessor] [DESKTOP-RQG9008] [electrical_grid_g1] Started writing results
[2020-07-07T16:58:39,626][WARN ][o.e.x.m.u.p.ResultsPersisterService] [DESKTOP-RQG9008] [electrical_grid_g1] failed to index after [1] attempts. Will attempt again in [50ms].
```

Reply:
Added check for spaces in empty string in ba0dede3f530a09fd5219b8642bd2c1ca91ae907